src/components/LifeItems.svelte

```diff
@@ -0,0 +1,92 @@
+<script>
+  import { fade, fly } from 'svelte/transition';
+  import ExpandableSlots from './Expandable/ExpandableSlots.svelte';
+  import { birthDate } from '../stores/birthDate';
+  import { getLifeCount } from '../helpers/calculations';
+  import Preview from './Preview.svelte';
+  import CellsContainer from './CellsContainer/CellsContainer.svelte';
+  import { getCellSize, getCellProps } from './CellsContainer/helpers';
+  import arrow from '../../assets/images/arrow-black.svg';
+  import { toCapitalize } from '../helpers/common';
+
+  export let name = 'years';
+
+  const widthMap = {
+    weeks: 'max-w-6xl',
+    months: 'max-w-5xl',
+    years: 'max-w-2xl'
+  };
+
+  let liveCount, lifeItems, lived, total, size;
+
+  $: {
+    if ($birthDate) {
+      liveCount = getLifeCount(new Date($birthDate));
+      lifeItems = liveCount[name];
+      [lived, total] = liveCount[name];
+      size = getCellSize(total);
+    }
+  }
+
+  const hClass = 'font-bold text-lg sm:text-2xl';
```

Comment:
didn't find usage of this const
btw, in most cases is better to have const values in the very beggining of the code